nestkernel/simulation_manager.cpp

```diff
@@ -387,12 +404,9 @@ nest::SimulationManager::resume_()
   LOG( M_INFO, "SimulationManager::resume", "Simulation finished." );
 }
 
-void
+size_t
 nest::SimulationManager::prepare_simulation_()
 {
-  if ( to_do_ == 0 )
-    return;
-
```

Comment:
This could never be reached, as the calling function already has a check for `t != 0`